### PR TITLE
readme, data, first pipeline with a simple experiment

### PR DESCRIPTION
**Pull request by krajasethupathy**
@JasonTam can you try installation to see if it works? will add more experiments to `model_pipeline.py`

### CODE REVIEW

File: model_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+def cfg():
+  penalty = 'l2'
+  fit_intercept = False
+  features = ['Fare','SibSp','Parch']
+  folds = 10
+
+@ex.automain
+def run(penalty, fit_intercept,features,folds):
+
+	train = pd.read_csv('data/train.csv')
```
Comment (krajasethupathy) [inline model_pipeline.py]: 4 Ingredients:

- train, train_partial, validation, test

**Comment by krajasethupathy** [comment]
- for each set of parameters -> new ingredient, and define config for that ingredient.
- ex: run experiment with variant training params A and variant validation params B

**Comment by krajasethupathy** [comment]
More Ingredients:

- **beefen** up the preprocessing ingredients. maybe try some real preprocessing and add variants for different kinds of missing data imputation approaches for the `age` variable. 
- **model or pipeline**: variants for each kind of model. i guess each named config can be a different set of steps to generate a specific model. can have a variable named `model` that is common between each variant. this will be _captured_ and loaded. 
- **validation**: variants for train/test split, kfold [?]
- what actually happens in the `capture`? this simple approach might be super slow/inefficient with larger dataframes

**Comment by JasonTam** [comment]
`capture` is just a convenience decorator to grab params from config. I'd assume the overhead in capturing params is negligible.

**Pull request merged**